repository_service_tuf/cli/artifact/download.py

```diff
@@ -0,0 +1,254 @@
+# SPDX-FileCopyrightText: 2023 VMware Inc
+#
+# SPDX-License-Identifier: MIT
+
+
+import base64
+import os
+import sys
+from hashlib import sha256
+from pathlib import Path
+from typing import Optional
+from urllib import request
+from urllib.parse import urlparse
+
+from click import Context
+from tuf.api.exceptions import DownloadError, RepositoryError
+from tuf.ngclient import Updater
+
+from repository_service_tuf.cli import click, console
+from repository_service_tuf.cli.artifact import artifact
+from repository_service_tuf.helpers.hash_bins import find_hash_bin
+
+
+def update_artifact_name_with_hash_prefix(artifact_name: str) -> str:
+    artifact_file_name = artifact_name.split("/")[-1]
+    hash_prefix = find_hash_bin(artifact_file_name)
+    new_artifact_file_name = hash_prefix + "-" + artifact_file_name
+    splitted = artifact_name.rsplit("/", 1)
+    if len(splitted) > 1:
+        return artifact_name.rsplit("/", 1)[0] + "/" + new_artifact_file_name
+    else:
+        return new_artifact_file_name
+
+
+def decode_trusted_root(root) -> str:
```

Comment:
What is the type of this `root`?

Reply:
`base64` is frequently causing issues with tox typing and this might cause unexpected CI failures in future and so be a blocker for future PRs. This is why I consider it safer to omit - it will anyway fail with a clear message if the type is improper. We talked about putting `any` instead, to just stick to our convention, but reconsidering, I don't think this is a good idea, as it reads as "root can be any", which is wrong. 
Given that the type restriction anyway falls down to the `base64` library without any additional logic from our side, the optimal solution is to just omit.